ibis/expr/types/numeric.py

```diff
@@ -198,7 +200,13 @@ def clip(
         --------
         >>> import ibis
         >>> ibis.options.interactive = True
-        >>> t = ibis.memtable({"values": range(8)})
+        >>> t = ibis.memtable(
```

Comment:
nit: can you just add some NULLs to the existing column of data? I think that shows all aspects of clip, and is less complicated and less lines